app/routes/application.js

```diff
@@ -197,18 +201,7 @@ export default Route.extend(
 				routerScroll.set('key', get(window, 'history.state.uuid'));
 				routerScroll.update();
 
-				// Because application controller needs wiki-page controller
-				// we can't be sure that media model will be ready when aplication controller is ready
 				run.scheduleOnce('afterRender', () => {
-					const file = controller.get('file'),
-						map = controller.get('map');
-
-					if (!isEmpty(file)) {
-						controller.openLightboxForMedia(file);
```

Comment:
so where this happens now?
what will open a lightbox when user opens a page with ?file=XYZ?

Reply:
https://wikia-inc.atlassian.net/browse/XW-4740